File: Practice(Beginner)/C++/Chef and Fruits.cpp
// Source: https://www.codechef.com/problems/FRUITS

#include <bits/stdc++.h>
using namespace std;

int main() {
    int t, apple, orange, coin;
    cin >> t;
    while(t--){
        cin >> apple >> orange >> coin;
        abs(apple-orange) <= coin ? cout << 0 << endl : cout << abs(apple-orange)-coin << endl;
    }
    return 0;
}

File: Practice(Beginner)/Python3/Chef_and_Fruits.py
# Source: https://www.codechef.com/problems/FRUITS

n = int(input())
for i in range(n):
    a,o,c = map(int,input().split())
    if(a>o and (a-o)>c):
        print(a-o-c)
    elif(a<o and (o-a)>c):
        print(o-a-c)
    else:
        print(0)
